Gordon360/Models/ViewModels/MembershipUploadViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Gordon360.Models.CCT
+{
+    public partial class MembershipUploadViewModel
+    {
+        public string ACTCode { get; set; }
+        public string SessCode { get; set; }
+        public string Username { get; set; }
+        public string PartCode { get; set; }
+        public string CommentText { get; set; }
+        public bool GroupAdmin { get; set; }
+        public bool Privacy { get; set; }
+        public static MembershipUploadViewModel FromREQUEST(REQUEST request)
```

Comment:
```suggestion

        public static MembershipUploadViewModel FromREQUEST(REQUEST request)
```

Reply:
I would prefer an implicit operator over this oddly-cased method.
```suggestion
        public static implicit operator MembershipUploadViewModel(REQUEST request)
```